app/models/peer_review.rb

```diff
@@ -29,4 +29,30 @@ def no_students_should_be_reviewer_and_reviewee
       end
     end
   end
+
+  # Gets the peer review between the groups, or returns nill if not found.
+  def self.get_peer_review_for(reviewer, reviewee)
+    results = Result.where(submission: reviewee.current_submission_used)
+    PeerReview.where(result: results, reviewer: reviewer).first
+  end
+
+  def self.does_review_exist_between?(reviewer, reviewee)
+    not PeerReview.get_peer_review_for(reviewer, reviewee).nil?
+  end
+
+  def self.able_to_assign_peer_review_to?(reviewer, reviewee)
+    return (not PeerReview.does_review_exist_between?(reviewer, reviewee)) &&
+        reviewer.does_not_share_any_students?(reviewee)
+  end
+
+  # Creates a new peer review between the reviewer and reviewee, otherwise if
```

Comment:
It would be helpful to mention that `reviewer` and `reviewee` are both groupings.